src/components/ProfilePreview.vue

```diff
@@ -77,19 +77,13 @@ export default {
   },
   methods: {
     closeProfile() {
-      const elementToReturnFocusTo = this.$store.state.profilePreview.lastTrigger;
-
-      this.$store.commit('updatePreviewProfileId', {
-        newId: null,
-        lastTrigger: null,
-      });
-
-      if (elementToReturnFocusTo) {
-        elementToReturnFocusTo.focus();
-      }
+      this.$router.push({ name: 'Orgchart' });
+      this.lastActive.focus();
+      this.lastActive = document.body;
     },
   },
   mounted() {
+    this.lastActive = document.activeElement || document.body;
```

Comment:
NIce . This makes sense. 

I think we don't need the `|| document.body` fallback, afaik `document.activeElement` always exists.